src/utils/format-numbers.js

```diff
@@ -25,12 +25,13 @@ import type { Microseconds, Milliseconds } from '../types/units';
 export function formatNumber(
   value: number,
   significantDigits: number = 2,
-  maxFractionalDigits: number = 3
+  maxFractionalDigits: number = 3,
+  style: string = 'decimal'
```

Comment:
nit: because the `style` can be either `'percent'` or `'decimal'`, this should be reflected in the Flow typing. Can you please replace this line by:
```js
style: 'decimal' | 'percent' = 'decimal' 
```